[2025-01-29T11:13:01.179+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-29T11:13:01.274+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main_dag.customer_Transform manual__2025-01-29T11:12:34.645629+00:00 [queued]>
[2025-01-29T11:13:01.313+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main_dag.customer_Transform manual__2025-01-29T11:12:34.645629+00:00 [queued]>
[2025-01-29T11:13:01.315+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 3
[2025-01-29T11:13:01.380+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): customer_Transform> on 2025-01-29 11:12:34.645629+00:00
[2025-01-29T11:13:01.577+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1764) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-29T11:13:01.603+0000] {standard_task_runner.py:72} INFO - Started process 1783 to run task
[2025-01-29T11:13:01.658+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'customer_Transform', 'manual__2025-01-29T11:12:34.645629+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp0qy492ir']
[2025-01-29T11:13:01.705+0000] {standard_task_runner.py:105} INFO - Job 34: Subtask customer_Transform
[2025-01-29T11:13:02.537+0000] {task_command.py:467} INFO - Running <TaskInstance: main_dag.customer_Transform manual__2025-01-29T11:12:34.645629+00:00 [running]> on host ae52aba2dca2
[2025-01-29T11:13:03.514+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='itsMe!!' AIRFLOW_CTX_DAG_ID='main_dag' AIRFLOW_CTX_TASK_ID='customer_Transform' AIRFLOW_CTX_EXECUTION_DATE='2025-01-29T11:12:34.645629+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-29T11:12:34.645629+00:00'
[2025-01-29T11:13:03.580+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-29T11:13:06.706+0000] {logging_mixin.py:190} INFO - <class 'pandas.core.frame.DataFrame'>
Index: 638 entries, 0 to 2586
Data columns (total 3 columns):
 #   Column         Non-Null Count  Dtype 
---  ------         --------------  ----- 
 0   Customer ID    638 non-null    object
 1   Customer Name  638 non-null    object
 2   Segment        638 non-null    object
dtypes: object(3)
memory usage: 19.9+ KB
[2025-01-29T11:13:06.709+0000] {logging_mixin.py:190} INFO - None
[2025-01-29T11:13:07.507+0000] {logging_mixin.py:190} INFO - !_upload_to_S3_complete_!
[2025-01-29T11:13:07.510+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-29T11:13:07.539+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-29T11:13:07.541+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=customer_Transform, run_id=manual__2025-01-29T11:12:34.645629+00:00, execution_date=20250129T111234, start_date=20250129T111301, end_date=20250129T111307
[2025-01-29T11:13:07.608+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-29T11:13:07.632+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
